book/version_control.md

```diff
@@ -395,140 +336,102 @@ To handle a data breach, you should:
-GitHub provides additional tools for collaborative workflows, including:
-* Access management for viewing and contributing
-* Issues
-* Project boards
-* Forking
-* Pull requests
-* Continuous Integration
+## GitHub
 
-Many of these project management tools are also discussed on the [GitHub features page](https://github.com/features/project-management/).
+A number of version control platforms extend the functionality of Git, to further improve collaborative working.
 
+Here we describe some of the beneficial features supported by [GitHub](https://github.com/), the world's leading software development platform. GitHub provides additional tools for better management of collaborative work. Many of these tools are also discussed in detail on the [GitHub features page](https://github.com/features/project-management/), but we will describe how they may be applied in analytical workflows here.
 
-### Access management
+### Efficient use of issues
 
-GitHub repositories have a [visibility setting](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/setting-repository-visibility) that represents whether the repo can be viewed publicly or only by owners of the project,
-Note that some GitHub features are limited or are not available for private repos on free GitHub accounts.
+Issues offer a method for requesting or recording tasks, including enhancements and bug fixes in a project. They act as a collaborative todo list, which users and developers can easily contribute to.
 
-Organizations provide an area for collating multiple repos that are associated with a particular team or department.
-Within Organizations, Teams can be also created to manage view and contribution permissions for projects within the Organization.
-External collaborators can also be added to projects, to allow direct contribution from those outside of the Organization.
-
-Despite the visibility status of a repo, only Organization members and collaborators may make direct contributions to the code in the repo.
-Others can contribute by Pull Request.
-
-Detailed setup and management of Organizations and Teams are described in the [relevant section of the GitHub Docs](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams).
+The basic elements of an issue are the:
+* Title and description, provided by the person that submitted the issue
+* Labels that categorise the issue (e.g. enhancement or bug)
+* Comments section where the issue can be discussed
+* Assigned developers that are working on resolving the issue
 
+Within an issue's description and comments, you can reference other issues both within (e.g. `#12`) and between repos, and tag individuals to notify them of your comments (`@whatstheirface`). Similarly, issues can be linked to specific [changes that will be merged to resolve or help to solve the issue](pull-requests). This makes them useful for discussing bugs and new features within a team.
```

Comment:
not only that you can also raise that if you reference issues in commit messages and have appropriately named branches, once you reference #12 on the commit message, github can close issues and do other stuff for you - that is worth teaching and embedding as best practice. 

I thin also stressing that its often a good idea to relate each explicit ticket that you're working on to a branch so you know that a given merge at this point in time solved a given github ticket (and github can auto close the issue). gimmie a shout if you wanna chat about this bit.